code/game/objects/items/devices/scanners.dm

```diff
@@ -1006,21 +1006,24 @@ GENE SCANNER
 		symptomholder.Finalize()
 		symptomholder.Refresh()
 		to_chat(user, "<span class='warning'>you begin isolating [chosen].</span>")
-		if(do_mob(user, AM, (600 / scanner.rating)))
-			create_culture(symptomholder, user)
-	else if(do_mob(user, AM, (timer / scanner.rating)))
-		create_culture(A, user)
+		if(do_mob(user, AM, (600 / scanner.rating + 1)))
+			create_culture(symptomholder, user, AM)
+	else if(do_mob(user, AM, (timer / scanner.rating + 1)))
+		create_culture(A, user, AM)
 
 /obj/item/extrapolator/proc/create_culture(var/datum/disease/advance/A, mob/user)
-	if(cooldown > world.time - (1200 / scanner.rating))
+	if(cooldown > world.time - (12 / scanner.rating))
```

Comment:
The changelog says the extrapolator no longer has cooldowns - seems like that's not true, but the cooldown has been reduced to a sub-second value instead?